packages/kubernetes-1.19/kubelet-config

```diff
@@ -93,6 +93,12 @@ topologyManagerPolicy: {{settings.kubernetes.topology-manager-policy}}
 {{#if settings.kubernetes.pod-pids-limit includeZero=true}}
```

Comment:
For the lack of a better place to post this comment: please group your changes in your commits, I think you could have just one since all the changes you made work as a unit. But I'm happy with two commits as well as long as the grouping makes sense.